native/app/screens/OwnReviews.tsx

```diff
@@ -0,0 +1,112 @@
+import { MaterialIcons } from '@expo/vector-icons';
+import { useNavigation } from '@react-navigation/native';
+import React, { ReactElement, useEffect, useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ListItem, Avatar } from 'react-native-elements';
+import { Badge } from 'react-native-elements/dist/badge/Badge';
+import { ScrollView } from 'react-native-gesture-handler';
+import HeaderWithBackButton from '~/components/organisms/HeaderWithBackButton';
+import { Auth } from '~/store/models';
+import { getOwnReviews } from '~/store/usecases';
+import { IReviewResponse } from '~/types/response';
+import { COLOR, FONT_SIZE } from '~~/constants/theme';
+
+const styles = StyleSheet.create({
+  badgeStyle: {
+    backgroundColor: COLOR.BACKGROUND_WHITE,
+    alignSelf: 'flex-start',
+    marginStart: 10,
+    marginTop: 10,
+    height: 30,
+    width: 150,
+    borderRadius: 75,
+  },
+  reviewListStyle: {
+    marginTop: 8,
+    marginBottom: 'auto',
+    height: '100%',
+  },
+  reviewStyle: {
+    flexDirection: 'column',
+    alignSelf: 'stretch',
+  },
+  titleStyle: {
+    fontSize: FONT_SIZE.LISTITEM_TITLE,
+    fontWeight: 'bold',
+    paddingBottom: 4,
+    marginBottom: 4,
+  },
+  subtitleStyle: {
+    fontSize: FONT_SIZE.LISTITEM_SUBTITLE,
+    paddingVertical: 4,
+  },
+});
+
+interface Props {
+  auth: Auth.Model;
+}
+
+const nullData: IReviewResponse = {
+  total: 0,
+  offset: 1,
+  limit: 1,
+  reviews: [
+    {
+      createdAt: '',
+      book: {
+        id: 0,
+        title: '',
+        thumbnailUrl: '',
+      },
+      impression: '',
+      id: 0,
+      updatedAt: '',
+    },
+  ],
+};
+
+export const OwnReviews = function OwnReviews(props: Props): ReactElement {
+  const [reviewList, setReviewList] = useState<IReviewResponse>(nullData);
+  const navigation = useNavigation();
+
+  useEffect(() => {
+    const f = async () => {
+      const res = await getOwnReviews(props.auth.id);
+      setReviewList(res.data);
+    };
+    f();
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    const f = async () => {
      const res = await getOwnReviews(props.auth.id);
      setReviewList(res.data);
    };
    f();
  });
```

変更検知しないなら、最後の `[]` はいらない...? (React詳しくないので間違ってたらごめんなさい🙇‍♂️)

Reply:
`[]`にしとくと、コンポーネントの初回描写時に関数を実行
`[]`を消すと、描写するたびに実行
っていう動きになるんだけど、どっちの方がいいんですかね…?

Reply:
なるほど!!そういう動きだったらいらないかも🤔